rhel8/templates/csv/packages_removed.csv

```diff
@@ -15,24 +15,19 @@ httpd
 iputils
 kernel-tools
 kexec-tools
-libcgroup
-libcgroup-tools
```

Comment:
@yuumasato why were these dropped? Packages still exist.

Reply:
They're mentioned that they are removed in the docs https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8-beta/html/8.0_beta_release_notes/changes_to_packages but they exist on my RHEL8 beta, so that means the docs are wrong.

Reply:
Since these packages are still shipped, I'll put them back into the list.